package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "project_cs372",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kaylebredner/Project_CS372.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kaylebredner/Project_CS372/issues"
+  },
+  "homepage": "https://github.com/kaylebredner/Project_CS372#readme",
+  "dependencies": {
+    "body-parser": "^1.20.1",
+    "express": "^4.18.2",
+    "https": "^1.0.0",
```

Comment:
I thought https and path were built in modules?

Reply:
I'm not sure. I included them in the npm install just in case.